From aa6a8850a97d5175197d32fe98cde82b4a7f2e39 Mon Sep 17 00:00:00 2001
From: Mianhan Liu <liumh1@shanghaitech.edu.cn>
Date: Thu, 30 Sep 2021 23:17:27 +0800
Subject: [PATCH] net/bridge/br_mrp_netlink.c: remove superfluous headers

br_mrp_netlink.c hasn't use any macro or function declared in net/genetlink.h.
Thus, these files can be removed from br_mrp_netlink.c safely without
affecting the compilation of the ./net module
---
 net/bridge/br_mrp_netlink.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/net/bridge/br_mrp_netlink.c b/net/bridge/br_mrp_netlink.c
index ce6f63c77..c0cbd4c80 100644
--- a/net/bridge/br_mrp_netlink.c
+++ b/net/bridge/br_mrp_netlink.c
@@ -1,6 +1,5 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
 
-#include <net/genetlink.h>
 
 #include <uapi/linux/mrp_bridge.h>
 #include "br_private.h"
-- 
2.25.1

